libr/include/r_types.h

```diff
@@ -654,4 +655,18 @@ static inline void r_run_call10(void *fcn, void *arg1, void *arg2, void *arg3, v
 # endif
 #endif
 
+// reference counter
+typedef void (*RFreeCallback)(void *a);
+typedef volatile int RRef;
+
+#define R_REF_NAME refcount
```

Comment:
why this define?

Reply:
to have asingle place to define the variable name

Reply:
is this important? we don't do this when we define a regular variable in a struct

Reply:
it's not a regular variable. its something we want to have in many structs and must be consistent. in case of conflicts i want to change that in 1 single place,